### ISSUE TITLE
Incapable of running `btcli run --cuda` on WSL ubuntu

### ISSUE THREAD

**Issue by urcades** [camfairchild/bittensor#897]
**Describe the bug**
I have followed the [bittensor installation guide](https://docs.bittensor.com/RunningAMiner.html) verbatim on a Windows machine running WSL. 

Everything was running perfectly, returning "working" results, up until I was asked to run `btcli run --cuda` to run bittensor on my GPU. 

The initial errors I received had to do with `cubit` not being  found as a dependency (patebin [here](https://pastebin.com/wCx35hx3)), so I ended up installing it from source in the /.bittensor directory and pip installing it accordingly. I made some progress, but after running the `btcli` command with the cuda flag, I continued to receive stack traces telling me that a `cubit.pyx` file was not being found in my home directory. After copying the same file from the cubit repository I cloned earlier to my home directory, I've hit a wall and a stack trace I can't seem to route around.

This is where I've landed:

https://pastebin.com/BXwsQxs6

**To Reproduce**
1. Follow the [installation guide ](https://docs.bittensor.com/RunningAMiner.html) verbatim
2. Run into errors after running `btcli run --cuda`

**Screenshots**
My first stack trace, saying cubit wasn't installed, even when I followed the installation steps successfully:

![1](https://user-images.githubusercontent.com/1195363/188055097-d8c29396-c58f-45a8-96ba-968f38f3c332.png)

My second stack trace, after installing cubit from source:

![2](https://user-images.githubusercontent.com/1195363/188055121-c67c1d8b-3c15-47cb-8b36-12af8a52bee7.png)

My third, intractable stack trace, saying I need an integer:

![3](https://user-images.githubusercontent.com/1195363/188055163-43f704ad-a854-4ec7-bbe6-634d9cb71391.png)

**Environment:**
 Windows 11 running WSL2:
 ```
 DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu 22.04.1 LTS"
```
 - Bittensor Version [e.g. 22]
 the latest release

**Additional context**

**Comment by urcades** [comment]
Wow, great timing:

After asking around in the discord for some help/insight, it turns out that running the btcli command with the following flags ends up working:

`btcli run --cuda --cuda.dev_id 0`

For whatever reason, omitting the `--cuda.dev_id 0` flag ends up with all of the aforementioned errors.

Using the proper flags results in a seemingly working GPU registration process. It's currently solving now.

**Comment by camfairchild** [comment]
This is fixed when #893 is merged. Only happens for CUDA registration using `btcli run` without specifying the `--cuda.dev_id` flag
